src/modules/adsupport.js

```diff
@@ -1615,6 +1615,9 @@ export default function (playerInstance, options) {
     };
 
     playerInstance.decreaseSkipOffset = () => {
+        if (typeof playerInstance.vastOptions === 'undefined' || playerInstance.vastOptions === null) {
```

Comment:
what happens if they are empty? it will break anyway

Reply:
This ticket can't be fixed by testing if something is undefined. You'll have to take a look at the try catch methods to see what the errors are, and based on that make a fix. The vastOptions could have data in it, but the data it gets from the server might be invalid xml, a 404/500/..., or maybe even not a xml at all.

Reply:
Updated the code